### PR TITLE
CIV-14878 CCD PR - Create Judgments folder in Case File

### PR DESCRIPTION
**Pull request by sjamihmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-14878

### Change description ###
Create Judgments folder in Case File

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/Categories/Categories.json [hunk 1]
```diff
@@ -1007,6 +1007,14 @@
     "DisplayOrder": "4",
     "ParentCategoryID": "orders"
   },
+  {
+    "LiveFrom": "05/12/2022",
+    "CaseTypeID": "CIVIL",
+    "CategoryID": "judgments",
+    "CategoryLabel": "Judgments",
+    "DisplayOrder": "6",
```
Comment (hmcts-version1-ignacio) [inline ccd-definition/Categories/Categories.json:1015]: Isn't DisplayOrder 5?

Comment (sjamihmcts) [reply ccd-definition/Categories/Categories.json:1015]: In Categories-SettleDiscontinue-nonprod.json, DisplayOrder as 5 is already given to discontinueNotices.

Comment (hmcts-version1-ignacio) [reply ccd-definition/Categories/Categories.json:1015]: Shouldn't this go in a non-prod file?

File: ccd-definition/Categories/Categories-Judgments-nonprod.json
Comment (hmcts-version1-ignacio) [inline ccd-definition/Categories/Categories-Judgments-nonprod.json:1]: We were using -JO-nonprod, right?

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: ccd-definition/Categories/Categories.json (hunk 1 above)
Comment (sjamihmcts) [reply ccd-definition/Categories/Categories.json]: Done

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**